tool/update_perfetto_assets.dart

```diff
@@ -0,0 +1,73 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+void main(List<String> args) {
+  final mainDevToolsDirectory = Directory.current;
+  if (!mainDevToolsDirectory.path.endsWith('/devtools')) {
```

Comment:
To make this more flexible what if we require a DEVTOOLS_DIR env var?

That way we can do fun stuff like adding the tools dir to our PATHs and not have to worry about our location:D

Reply:
discussed offline - we can optimize this in the future if needed. This script should never really be called by a human anyway - it should be called by the update_perfetto.sh script